### PR TITLE
upgrade minio-operator version to v3.0.13

### PR DESCRIPTION
**Pull request by soulseen**
Signed-off-by: Xiaoyang Zhu <zhuxiaoyang1996@gmail.com>

fix https://github.com/goharbor/harbor-cluster-operator/issues/153

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,22 +3,23 @@ module github.com/goharbor/harbor-cluster-operator
...
+	k8s.io/apimachinery v0.19.0-rc.3
 	k8s.io/client-go v11.0.0+incompatible
-	sigs.k8s.io/controller-runtime v0.6.0
+	k8s.io/kube-openapi v0.0.0-20200805222855-6aeccd4b50c6 // indirect
+	k8s.io/utils v0.0.0-20200729134348-d5654de09c73 // indirect
+	sigs.k8s.io/controller-runtime v0.6.1-0.20200804124940-17eebbff0d48
 )
 
-replace k8s.io/client-go v11.0.0+incompatible => k8s.io/client-go v0.18.2
+replace k8s.io/client-go v11.0.0+incompatible => k8s.io/client-go v0.0.0-20200813012017-e7a1d9ada0d5
```
Comment (steven-zou) [inline go.mod:25]: It's a little weird, `v0.0.0-`?

Comment (steven-zou) [reply go.mod:25]: a dev build?

Comment (soulseen) [reply go.mod:25]: this is to use a specific latest commit version, like `github.com/minio/operator v0.0.0-20200814200655-60bf757aac60`.

Comment (soulseen) [reply go.mod:25]: > a dev build?

yes, but I think this is the right way to resolve conflicts, like many other repos, such as `https://github.com/kubernetes/client-go/blob/master/go.mod#L38`.

Comment (steven-zou) [reply go.mod:25]: ok

**Pull request merged**